widgetsnbextension/src/manager.js

```diff
@@ -593,12 +593,12 @@ WidgetManager.prototype.deleteSnapshots = function() {
     }).bind(this));
 };
 
-WidgetManager.prototype._create_comm = function(comm_target_name, model_id, metadata) {
+WidgetManager.prototype._create_comm = function(comm_target_name, model_id, data) {
     var that = this;
     return this._get_connected_kernel().then(function(kernel) {
-        if (metadata) {
-            return kernel.comm_manager.new_comm(comm_target_name, {},
-                                                that.callbacks(), metadata, model_id);
+        if (data) {
+            return kernel.comm_manager.new_comm(comm_target_name, data,
+                                                that.callbacks(), {}, model_id);
```

Comment:
@SylvainCorlay Hmm... not sure I understand the backwards incompatibility wrt jupyter-js-widgets. That code only implements a stub. I do see a problem if the widgetsnbextension and ipywidgets get out of sync.

I'm fine with making the change. How about a target version for removal? Or does it even matter?